tests/integration/base_block_test.go

```diff
@@ -221,6 +240,26 @@ func retryBlockImageCountCheck(helper *clients.TestClient, imageCount, expectedC
 	return false
 }
 
+func retryPVCheck(k8sh *utils.K8sHelper, name string, exists bool, status string) bool {
+	inc := 0
+	for inc < utils.RetryLoop {
+		pv, err := k8sh.GetPV(name)
+		if err != nil {
```

Comment:
To check if it doesn't exist, it's more reliable to call `errors.IsNotFound(err)`. There could be false positives if you check for any error.

Reply:
@travisn From what I understand, the way the `k8sh` helper functions are written, e.g., https://github.com/rook/rook/blob/6fd4d04f3b2df151514748900381e3200fa3a110/tests/framework/utils/k8s_helper.go#L1036

it does not return the original `err` from the `k8sh.Clientset.CoreV1...`, but creates a new error using `fmt.Errorf`, and so using `errors.IsNotFound` here always returns `false`.

Reply:
hmm, the helper should be made smarter about this, but this is fine for now.